doc/Quick-Start-Guide.rst

```diff
@@ -176,6 +192,8 @@ Troubleshooting
         
 Tested by:
 
+- Dec 1st, 2020: Huang Hua (hua.huang@seagate.com) in CentOS 7.7.1908.
```

Comment:
Dec 1, 2020. Please remove the full stop.

Reply:
Thank you, fixed.

Reply:
Thank You